### ISSUE TITLE
Not working on us-east-1

### ISSUE THREAD

**Issue by OpNop** [leanderlee/ec2-sg-notes#1]
I changing the matches in the manifest to `"matches": ["https://*.console.aws.amazon.com/ec2/*"],` and that got the script to load but the table classes/ids are all different. One big change I see is that all the tabs have the same class (can only test on us-east-1) `.N2F.gwt-TabLayoutPanelTab`

Hope to see this plugin succeed, because there is no sign of AWS getting this implemented any time soon.

**Comment by leanderlee** [comment]
Right. You would have to go in and change these classes until we come up with a better solution --

https://github.com/leanderlee/ec2-sg-notes/blob/master/content.js#L2-L8
